### DISCUSSION TITLE
It'll be better to tag the repo with the list name if it belongs to any list in the "starred repo" page

### DISCUSSION THREAD

**Discussion by 1c383r9** [community/community#59059] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Currently we have to go into a list to inspect the repos, because the list name that the repo belongs to isn't displayed on the starred repo page(https://github.com/${USERNAME}?tab=stars).

Thx.

**Comment by ysh83737** [comment]
I think this is a good proposal, and I also hope that I can know which list this repo belongs to.

Maybe it looks like this:
<img width="925" alt="image" src="https://github.com/community/community/assets/32612643/59e28582-b4f5-43c8-a646-d3b254bc18ed">

**Comment by Rastiplan** [comment]
https://docs.github.com/ru/get-started/getting-started-with-git/managing-remote-repositories?platform=windows

**Reply by danmaster01** [reply]
👍👍
